HISTORY.rst

```diff
@@ -6,6 +6,12 @@ Release History
 unreleased
 +++++++++++++++
 
+**IoT Hub Updates**
```

Comment:
Should be under `0.17.0` (upcoming next release)

Reply:
Makes sense, will update.

Reply:
Done